patterns/index-writer.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Title: Writer Index Template
+ * Slug: twentytwentyfour/index-writer
+ * Template Types: index
```

Comment:
I think we need to add page and home here too

Reply:
wp_template?

Reply:
that would be Post Type instead of Template Type, maybe that's what we are missing, but I see other similar patterns don't have it? maybe those need it as well... I'm already confused with all the back and forth

Reply:
Me too